docs/UserGuide.md

```diff
@@ -143,6 +143,45 @@ To aid the user, this command starts the interactive tutorial for either the [Ci
 
 * `TYPE` can be either `circuit` or `boolean`.
 
+Once the user has entered the interactive tutorial, they are to follow the instructions provided to them as follows:
```

Comment:
Since this is the user guide, you can consider changing the instances of "the user" to "you" instead. The rest of the document does not use "the user".

Reply:
Sure, I have changed it :)